lighthouse-core/gather/gatherers/fonts.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer');
+const fontUrlRegex = new RegExp('url\\((?:"|\')([^"]+)(?:"|\')\\)');
```

Comment:
hm this is just getting the URL as its defined then isn't it? Ideally we could get the resolved URL, but not sure best way.

Perhaps just do this in the page and set the href of a link to get the real value?

Reply:
I needed this to parse
```
@font-face {
font-family: test;
src: local(Test), url('test.woff2'), url("test.woff");
```

Reply:
yeah totally get why it's needed just hoping we can report `https://example.com/test.woff2` instead of just `test.woff2` 👍